README.md

```diff
@@ -64,12 +64,14 @@ If you are working on Windows or you don't want to install the dependencies on y
       - ILI9341
       - ST7735
       - SH1107
+      - SSD1306(I2C)
```

Comment:
If this is a I2C device, then lets place it under the I2C section. Maybe we should reorganize the entire tree based on type instead of protocol type....

Reply:
Are you saying do something like this
spi 
 |- display
 | - sensors

i2c
 |- display
 | sensors


etc?